modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaClientCodegen.java

```diff
@@ -576,7 +576,8 @@ public void processOpts() {
             // The flag below should be set for all Java libraries, but the templates need to be ported
             // one by one for each library.
             supportsAdditionalPropertiesWithComposedSchema = true;
-
+            setUseJakartaEe(true);
+            applyJavaxPackage();
```

Comment:
Hmm... if you did `setUseJakartaEe()` before the `super` call, then you wouldn't also need to call `applyJavaxPackage()`. But then you'd need to duplicate the if statement logic... maybe just some documentation?

```java
// When using `setJakertaEe()` you should also call `applyJavaxPackage()` to capture the property value.
```

Reply:
@welshm I've slightly changed approach, please review one more time